thirdparty/libwebsockets/LICENSE

```diff
@@ -33,19 +33,21 @@ to get original sources with the liberal terms.
 
 Original liberal license retained
 
-  - lib/sha-1.c         - 3-clause BSD license retained, link to original
+  - lib/misc/sha-1.c    - 3-clause BSD license retained, link to original
```

Comment:
We also have a copy of their license text in `COPYRIGHT.txt`, so you'd need to make those refactor changes in there too. Maybe not the weird `` characters that they put in the original LGPL-2.1 text though.

Reply:
> We also have a copy of their license text in COPYRIGHT.txt

@akien-mga done :)